[2025-01-25T19:11:54.166+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-25T19:11:54.186+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prim_pipelines.get_data_from_prim_api manual__2025-01-25T18:48:55.829990+00:00 [queued]>
[2025-01-25T19:11:54.197+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prim_pipelines.get_data_from_prim_api manual__2025-01-25T18:48:55.829990+00:00 [queued]>
[2025-01-25T19:11:54.198+0000] {taskinstance.py:2866} INFO - Starting attempt 7 of 7
[2025-01-25T19:11:54.212+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): get_data_from_prim_api> on 2025-01-25 18:48:55.829990+00:00
[2025-01-25T19:11:54.225+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=518) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-25T19:11:54.227+0000] {standard_task_runner.py:72} INFO - Started process 533 to run task
[2025-01-25T19:11:54.227+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'prim_pipelines', 'get_data_from_prim_api', 'manual__2025-01-25T18:48:55.829990+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpveiqx4sh']
[2025-01-25T19:11:54.228+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask get_data_from_prim_api
[2025-01-25T19:11:54.279+0000] {task_command.py:467} INFO - Running <TaskInstance: prim_pipelines.get_data_from_prim_api manual__2025-01-25T18:48:55.829990+00:00 [running]> on host c96922973881
[2025-01-25T19:11:54.366+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='prim_pipelines' AIRFLOW_CTX_TASK_ID='get_data_from_prim_api' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T18:48:55.829990+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T18:48:55.829990+00:00'
[2025-01-25T19:11:54.367+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-25T19:11:54.846+0000] {python.py:240} INFO - Done. Returned value was: dags/data/stations.json
[2025-01-25T19:11:54.880+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-25T19:11:54.881+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=prim_pipelines, task_id=get_data_from_prim_api, run_id=manual__2025-01-25T18:48:55.829990+00:00, execution_date=20250125T184855, start_date=20250125T191154, end_date=20250125T191154
[2025-01-25T19:11:54.927+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-25T19:11:54.965+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-25T19:11:54.967+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
